testsuite/features/support/commonlib.rb

```diff
@@ -219,6 +219,21 @@ def get_client_type(name)
   end
 end
 
+def client_public_ip(host)
+  node = get_target(host)
+  interfaz = 'eth0'
```

Comment:
Instead of having this initial value, I would have an `else` the `case` statement.